src/index.html

```diff
@@ -6,9 +6,31 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="index.html">
+          <img src="images/logo.png" alt="moyo logo">
+        </a>
+      </div>
+      <nav class="nav_menu">
+        <ul>
+          <li class="link"><a class="is-active" href="#">Apple</a></li>
+          <li class="link"><a href="#">Samsung</a></li>
+          <li class="link"><a href="#">Smartphones</a></li>
+          <li class="link">
+            <a class="data-test"
+            href="#" data-qa="hover">Laptops & Computers</a>
```

Comment:
code styling from task readme.md 
```suggestion
            <a 
              class="nav__linkt"
              href="#" data-qa="hover"
            >
              Laptops & Computers
            </a>
```